app/views/overview/_service-instance-row.html

```diff
@@ -6,8 +6,8 @@
     <div class="list-pf-content">
       <div class="list-pf-name">
         <h3>
-          <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
-          <div ng-bind-html="row.apiObject.metadata.name | highlightKeywords : row.state.filterKeywords" class="list-row-longname"></div>
```

Comment:
Done. BTW: running `grunt build` changes `dist/scripts/vendor.js` and `app/index.html` in ways that aren't the result of my changes. That's why I haven't committed them. (They also change if I perform the build without my changes - on master).